network_provisioning/README.md

```diff
@@ -0,0 +1,5 @@
+# Network Provisioning component
+
+[![Component Registry](https://components.espressif.com/components/espressif/network_provisioning/badge.svg)](https://components.espressif.com/components/espressif/network_provisioning)
+
+This component extends [wifi_rovisioning component](https://github.com/espressif/esp-idf/tree/master/components/wifi_provisioning) to encompass broader network provisioning, including Thread provisioning. Both the Wi-fi devices and Thread devices can be provisioned into network with the esp_prov.py script.
```

Comment:
The link will disappear after we remove `wifi-provisioning` component from IDF.

Referred to [wifi-provisioning](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/provisioning/wifi_provisioning.html#wi-fi-provisioning), suggest the following introduction:

```
The network provisioning component provides APIs that control the network provisioning service for receiving and configuring network credentials via secure [Protocol Communication](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/provisioning/protocomm.html) sessions.

It currently supports both Wi-Fi and Thread network provisioning:
- Provision Wi-Fi credentials over SoftAP or Bluetooth LE
- Provision Thread credentials over Bluetooth LE
```

Reply:
Done